src/evaluator/__tests__/evaluator.ts

```diff
@@ -240,3 +240,42 @@ describe('basic lambda expressions', () => {
     expectJsonReadEvalError([['lambda', ['f'], ['f', 1], 'f'], 1], test_env());
   });
 });
+
+describe('basic cond expressions', () => {
+  test('valid', () => {
+    expectJsonReadEvalPrint(['cond', [true, 1]], test_env()).toMatchInlineSnapshot(`1`);
+
+    expectJsonReadEvalPrint(['cond', [true, 1], [true, 2]], test_env()).toMatchInlineSnapshot(`1`);
+
+    expectJsonReadEvalPrint(['cond', [false, 1], [true, 2]], test_env()).toMatchInlineSnapshot(`2`);
+
+    expectJsonReadEvalPrint(['cond', [true, 1, 2], [true, 3]], test_env()).toMatchInlineSnapshot(
+      `2`
+    );
+
+    expectJsonReadEvalPrint(['cond', [false], [true], [true, 1]], test_env()).toMatchInlineSnapshot(
+      `true`
+    );
```

Comment:
Oooh interesting. This is the returning `test-expr` as per https://docs.racket-lang.org/reference/if.html right?

Reply:
yeah i think